src/styles/index.scss

```
@@ -1,3 +1,65 @@
+@import './variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+}
+
+.calendar__day {
+  display: flex;
+  position: relative;
+  height: $day-size;
+  width: $day-size;
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  border: solid $border-size black;
+  justify-content: center;
+  align-items: center;
+  transition:
+    background-color $animation-duration,
+    transform $animation-duration;
+
+  &:hover {
+    cursor: pointer;
+    transform: translateY(-20px);
+    background-color: $hover-color;
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+@each $day, $margin in (mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6)
```

Move it `(mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6)` to the variable and use it here
```suggestion
@each $day, $margin in $days
```